lookup.js

```diff
@@ -106,7 +115,10 @@ exports.init = function (sbot, config) {
 
     // Update roots
     if (!detailsLookup.has(metafeed)) {
-      detailsLookup.set(metafeed, null)
+      // !detailsLookup.has(metafeed) !== "metafeed is a root)
+      // this assumes that we have replicated the metafeed tree in the order
+      // that it was created.
```

Comment:
:fire: spotted a potential edge case that could cause us pain later.
Documented in code. Should this be an issue?